mycore-base/src/main/resources/config/mycore.properties

```diff
@@ -398,6 +398,7 @@ MCR.ConfigurationDirectory.template.files=mycore.properties,resources/META-INF/p
 ##############################################################################
 
 MCR.Jersey.resource.packages=org.mycore.frontend.jersey.resources,org.mycore.services.queuedjob
+MCR.Jersey.resource.urlPattern=rsc
```

Comment:
urlPattern is misleading and "resource" needs to be with capital letter 'R'. Suggest: `MCR.Jersey.Resource.ApplicationPaths=rsc`